### ISSUE TITLE
Increase checkbox spacing to have a 44px click handler

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/server#10617]
From https://github.com/nextcloud/server/issues/10094:

> Padding for checkbox settings (bottom left Files settings, Files sidebar link share options, Sharing settings): every entry needs at least 44px height for clickability, and more whitespace between checkbox and text
> Sharing: More padding for checkbox entries, 44px as usual

We should add a padding to all checkboxes with their labels in the global input css, but I would suggest to do this for 15, since i would expect this might cause quite some bugs all over the places where the checkboxes are used.

**Comment by MorrisJobke** [comment]
@nextcloud/designers Should we still do this? Then we would need to do it until Wednesday (it should be merged by then).

**Comment by MorrisJobke** [comment]
This seems similar to #7481

**Comment by MorrisJobke** [comment]
Moved to 16.

**Comment by skjnldsv** [comment]
I guess we could start with https://github.com/nextcloud/nextcloud-vue/pull/649 ?

**Comment by juliushaertl** [comment]
We're back to smaller click targets but most of it is handled by the library today. Closing.
